### PR TITLE
Tweak ping device_tracker docs

### PR DESCRIPTION
**Pull request by ronaldevers**
## Proposed change

I've clarified the ping integration docs a little bit.  I could not figure out from the `device_tracker` example and the text that `hostname` (under `hosts`) was meant to be the device name. Also, the type is wrong, `hosts` is not a list but a map.

So based on the current docs I was trying to do this in my config which would not work:

```
device_tracker:
  - platform: ping
    hosts:
      - hostname: 192.168.2.10
      - hostname: 192.168.2.11
```

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/ping.markdown [hunk 1]
```diff
@@ -86,14 +86,15 @@ To use this presence detection in your installation, add the following to your `
 device_tracker:
   - platform: ping
     hosts:
-      hostname: 192.168.2.10
+      hostname1: 192.168.2.10
+      hostname2: 192.168.2.11
```
Comment (frenck) [inline source/_integrations/ping.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters that they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/ping.markdown (hunk 1 above)
Comment (ronaldevers) [reply source/_integrations/ping.markdown]: Reading the docs again is says this is a map of device name to hostname or IP address. So maybe instead we make it:

```
hosts:
    device_name_1: 192.168.2.10
```

What do you think?

Comment (ronaldevers) [reply source/_integrations/ping.markdown]: Updated and added one more commit to fix the link to the people-tracking documentation.

**Review by frenck — APPROVED** [review]
Thanks, @ronaldevers 👍 

Happy New Year 🥂

**Pull request merged**